src/Files.App/Data/Items/WidgetFileTagCardItem.cs

```diff
@@ -1,47 +1,64 @@
 ﻿// Copyright (c) 2023 Files Community
 // Licensed under the MIT License. See the LICENSE.
 
-using Files.App.UserControls.Widgets;
-using Files.Core.Storage;
 using Files.Core.Storage.Extensions;
 using Files.Shared.Utils;
+using System.Windows.Input;
 
-namespace Files.App.ViewModels.Widgets
+namespace Files.App.Data.Items
 {
-	public sealed partial class FileTagsItemViewModel : WidgetCardItem
+	public sealed partial class WidgetFileTagCardItem : WidgetCardItem
 	{
+		// Fields
+
 		private readonly IStorable _associatedStorable;
 
 		// A workaround for lack of MVVM-compliant navigation support.
 		// This workaround must be kept until further refactor of navigation code is completed.
 		private readonly Func<string, Task> _openAction;
 
-		[ObservableProperty]
+		// Properties
+
+		public bool IsFolder
+			=> _associatedStorable is IFolder;
+
 		private IImage? _Icon;
+		public IImage? Icon
+		{
+			get => _Icon;
+			set => SetProperty(ref _Icon, value);
+		}
 
-		[ObservableProperty]
```

Comment:
A question to keep up to date with the current codebase: we're getting rid of the `[ObservableProperty]` attribute?

Reply:
@yaira2 Are you willing to use those attaributes including `[RelayCommand]`?